lib/eml2html/converter.rb

```diff
@@ -94,7 +122,13 @@ def read_attachments
     def replace_images_src(html)
       html.gsub(/(?<=src=['"])cid:[^'"]+(?=['"])/) do |match|
         cid = match.sub(/^cid:/, '')
-        @attachments.find{|a| a.cid == cid}.name
+        @cids_in_body.push(cid)
+        attachment = @attachments.find{|a| a.cid == cid }
```

Comment:
If you are using new Ruby you can use save navigator operator:
@attachments.find{|a| a.cid == cid }&.name || ''

Reply:
that's pretty cool, and yes, we're using a new enough ruby to take advantage of it, so I'll make the change.

Reply:
are you sure returning nil inside the gsub closure won't cause any bugs?

Reply:
Yes, `nil` is fine, just tested it:
```
> "asd".gsub(/./) { nil }
 => ""
```